### DISCUSSION TITLE
Cannot see most of `Insights` with GitHub Pro in private fork of a private orgainzation repo

### DISCUSSION THREAD

**Discussion by Young-Lord** [community/community#65161] [Repositories]
### Select Topic Area

Bug

### Body

Reproduce:
1. Create a organization name `org`
2. Create a private repo `org/repo`, navigate to `Insights`, most of the features are disabled with hint `Upgrade to GitHub Team or ... to enable`
3. Fork `org/repo` to your account with GitHub Pro, name the new repo `me/repo`
4. Navigate to `Insights`, most of the features are **disabled with hint `Upgrade to GitHub Pro or ... to enable` even if I already have GitHub Pro (part of GitHub Student Developer Pack)**

**Comment by arafajs** [comment]
You cannot see most of the Insights features in a private fork of a private organization repository, even if you have a GitHub Pro account. This is because Insights features are only available for public repositories and forks of public repositories.
